CHANGELOG.md

```diff
@@ -2,6 +2,324 @@
 - [v1.0.0 - v1.9.10](CHANGELOG-pre-v1.10.md)
 - [v0.11.6 and earlier](CHANGELOG-v0.md)
 
+## 1.16.0-rc2
```

Comment:
It's maybe a little counterintuitive but historically we replace the RC changelogs with the GA changelog, so I think the better thing to do here is to replace the RC1 content with the RC2 content.

Reply:
Ah. I had no idea on that, I can delete the RC1 content and just leave the RC2